editor-components/pricing-table/pricing-table6/pricing-table6.tsx

```diff
@@ -1392,47 +1179,31 @@ class PricingMultipleTwo extends BasePricingTable {
     const plans = this.castToObject<Pricing[]>("plans");
     const activeTab = this.getComponentState("activeTab");
-    const hasPlans =
-      Array.isArray(plans) &&
-      plans.length > 0 &&
-      activeTab !== undefined &&
-      plans[activeTab];
+    const hasPlans = Array.isArray(plans) && plans.length > 0 && activeTab !== undefined && plans[activeTab];
     const plan = hasPlans ? plans[activeTab].product : [];
     const planIndex = this.getComponentState("activePlan");
 
+    const rightItemExist = plan[planIndex]?.right_items?.list.length > 0;
+
     return (
-      <div className={this.decorateCSS("container")}>
-        <div className={this.decorateCSS("max-content")}>
-          <div className={this.decorateCSS("page")}>
-            <div className={this.decorateCSS("up-page")}>
-              {this.castToString(this.getPropValue("badge")) && (
-                <div className={this.decorateCSS("badge")}>
-                  {this.getPropValue("badge")}
-                </div>
-              )}
-              {this.castToString(this.getPropValue("title")) && (
-                <h1 className={this.decorateCSS("title")}>
-                  {this.getPropValue("title")}
-                </h1>
-              )}
-            </div>
+      <Base.Container className={this.decorateCSS("container")}>
+        <Base.MaxContent className={this.decorateCSS("max-content")}>
+          <Base.VerticalContent className={this.decorateCSS("page")}>
+            <Base.VerticalContent className={this.decorateCSS("up-page")}>
+              {this.castToString(this.getPropValue("badge")) && <Base.SectionSubTitle className={this.decorateCSS("badge")}>{this.getPropValue("badge")}</Base.SectionSubTitle>}
+              {this.castToString(this.getPropValue("title")) && <Base.SectionTitle className={this.decorateCSS("title")}>{this.getPropValue("title")}</Base.SectionTitle>}
+            </Base.VerticalContent>
             {hasPlans && (
               <div className={this.decorateCSS("plan-items")}>
                 {plans.map((plan: any, index: number) => {
                   const title2 = this.castToString(plan.getPropValue("title"));
-                  return title2 ? (
-                    <div
-                      className={
-                        this.decorateCSS("plan-title") +
-                        " " +
-                        (this.getComponentState("activeTab") === index &&
-                          this.decorateCSS("activeTab"))
-                      }
-                      onClick={() => this.setActiveTab(index)}
-                    >
-                      {title2}
-                    </div>
-                  ) : null;
+                  return (
+                    title2 && (
+                      <Base.P className={this.decorateCSS("plan-title") + " " + (this.getComponentState("activeTab") === index && this.decorateCSS("activeTab"))} onClick={() => this.setActiveTab(index)}>
+                        {title2}
```

Comment:
should use plan.getPropValue("title"). if not the inline editor will not appear on click